api/src/integrations/core/datasource.js

```diff
@@ -525,7 +525,7 @@ class CoreDataSource {
           _sourceRefCoreCollectionId: amendment.permit_amendment_guid,
           project: mineRecord._id,
           name: amendment.description,
-          date: amendment.issue_date,
+          date: new Date(amendment.issue_date),
```

Comment:
This needs checking if `issue_date` isn't null, otherwise you create EPOCH dates.